skglm/solvers/pn_solver.py

```diff
@@ -0,0 +1,295 @@
+import numpy as np
+from scipy.sparse import issparse
+from numba import njit
```

Comment:
really cosmetic: inside each import block we put shorter import first otherwise it summons @tommoral

Reply:
:smiling_imp: